packages/grafana-ui/src/components/DataLinks/DataLinkInput.tsx

```diff
@@ -151,10 +153,27 @@ export const DataLinkInput: React.FC<DataLinkInputProps> = memo(
       stateRef.current.onChange(Plain.serialize(editor.value));
     };
 
+    const getCharactersAroundCaret = () => {
+      const input: HTMLSpanElement | null = document.getElementById('data-link-input')!;
+      let precedingChar = '',
+        sel: Selection | null,
+        range: Range;
+      if (window.getSelection) {
+        sel = window.getSelection();
+        if (sel && sel.rangeCount > 0) {
+          range = sel.getRangeAt(0).cloneRange();
+          range.collapse(true);
```

Comment:
I would add
```suggestion
          // If the content is focused and editable, the caret will blink there.
          range.collapse(true);
```
because it's a bit out of the blue and not too widely used imo.

Reply:
good call. thanks!